middlewares.ts

```diff
@@ -72,19 +94,32 @@ export const authorizePls = async (
                 (v, i, self) =>
                     i === self.findIndex((t) => t.identifier === v.identifier)
             );
-
-        httpContext.set("user", {
-            ...user,
-            pls: plsResponse.data,
-            mandates,
-            groups,
-        });
-
-        next();
     } catch (err) {
-        res.sendStatus(500);
-        return;
+        const dfunktError = err as AxiosError;
+        if (dfunktError.response) {
+            const { status, data } = dfunktError.response;
+            // If status is 404, this user is simply not a dFunkt and the page
+            // should proceed normally. Otherwise, return an error.
+            if (status != 404) {
+                res.status(status).send(data);
+                return;
+            }
+        } else {
+            res.status(502).send(
+                "Bad Gateway - No response from the dFunkt API."
+            );
+            return;
```

Comment:
The else case here could also mean that `err` was some other error. Perhaps it would be nice to add something like
```js
if (!(err instanceof AxiosError)) {
  console.error(err);
  return res.sendStatus(500);
}
```
at the beginning of the `catch`?

Reply:
I just made a commit adding your proposal, thanks for pointing this out.

TypeScript wouldn't allow me to use `!(err instanceof AxiosError)`. Apparently `AxiosError` is not a type, but an interface. Luckily they added an `isAxiosError` flag to it, so I used that instead.

To be more consistent with the other error responses, I put the error message in `res.status(500).send(ERROR_MESSAGE)` instead.